utils/lib/wb-image-update/fit/install_update.sh

```diff
@@ -1203,10 +1219,12 @@ maybe_factory_reset() {
         fi
 
         rm -rf /tmp/empty && mkdir /tmp/empty
-        rsync -a --delete --exclude="/.wb-restore/" --exclude="/.wb-update/" /tmp/empty/ /mnt/data/
 
-        FACTORY_FIT_DIR="/mnt/data/.wb-restore"
-        FACTORY_FIT="${FACTORY_FIT_DIR}/factoryreset.fit"
+        local cmd=(rsync -a --delete --exclude="/.wb-restore/" --exclude="/.wb-update/" /tmp/empty/ /mnt/data/)
+        "${cmd[@]}" || (reset_immutable && "${cmd[@]}")
+
+        local FACTORY_FIT_DIR="/mnt/data/.wb-restore"
+        local FACTORY_FIT="${FACTORY_FIT_DIR}/factoryreset.fit"
```

Comment:
Чтобы не путать с глобальными переменными:
```suggestion
        local factory_fit_dir="/mnt/data/.wb-restore"
        local factory_fit="${FACTORY_FIT_DIR}/factoryreset.fit"
```

Reply:
поправил